core/api/src/graphql/admin/schema.graphql

```diff
@@ -196,6 +203,13 @@ type LightningPayment {
   status: LnPaymentStatus
 }
 
+type LnInvoice {
+  paymentHash: PaymentHash!
```

Comment:
no status?

Reply:
I will add status now that I have refactored the types